### PR TITLE
add php 8 support

### PR DESCRIPTION
**Pull request by ignacioalles**
Allow php version 8. Have to update version of dependant library `goetas-webservices/xsd2php` too. Issue #12

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		],
     "license": "MIT",
     "require": {
-        "php": "^7.2",
+        "php": ">=7.2",
```
Comment (Skyree) [inline composer.json:15]: I'd recommend rather `^7.2|^8.0` so this release of the component won't be usable with future major version of php that could be BC break

**Comment by miqwit** [comment]
Hello. Thanks for you contributions/suggestion. Give me bit more time to test it locally and approve it!

**Comment by miqwit** [comment]
In fact, I can't upgrade without upgrading `goetas-webservices/xsd2php` indeed. I don't see any plans from them to do so from now, so I am a bit stuck. It's worth interacting with them, and maybe suggest a pull request to migrate to PHP 8 so I can migrate as well. I will try to do it, but I encourage you to do so as well.

**Comment by ignacioalles** [comment]
@miqwit `goetas-webservices/xsd2php` is already upgraded in this PR. It seems they added php 8 support since version 0.4.2.

**Comment by miqwit** [comment]
It is not very clear that they support PHP 8. However, from this `travis.yml` file, they seem to support it :) https://github.com/goetas-webservices/xsd2php/compare/0.4.3...0.4.4#

I need more tries then, and I will update accordingly.

**Comment by miqwit** [comment]
I did that in another branch. Thanks for your contribution!

**Pull request closed without merging**